#! /bin/sh
patch -p1 -l -f $* < $0
exit $?

2014-08-04  Andre Heinecke  <aheinecke@intevation.de>

    Apply gnulib commit 8672534
    * gl/msvc-inval.c: Use __cdecl instead of cdecl
    * lib/gl/msvc-inval.c: Wtf, they have it twice o.O


diff -ru a/gl/msvc-inval.c b/gl/msvc-inval.c
--- a/gl/msvc-inval.c	2014-08-04 20:41:03.938286457 +0200
+++ b/gl/msvc-inval.c	2014-08-04 20:41:40.138286520 +0200
@@ -28,7 +28,7 @@

 # if MSVC_INVALID_PARAMETER_HANDLING == DEFAULT_HANDLING

-static void cdecl
+static void __cdecl
 gl_msvc_invalid_parameter_handler (const wchar_t *expression,
                                    const wchar_t *function,
                                    const wchar_t *file,
@@ -45,7 +45,7 @@

 #  if defined _MSC_VER

-static void cdecl
+static void __cdecl
 gl_msvc_invalid_parameter_handler (const wchar_t *expression,
                                    const wchar_t *function,
                                    const wchar_t *file,
@@ -94,7 +94,7 @@
     }
 }

-static void cdecl
+static void __cdecl
 gl_msvc_invalid_parameter_handler (const wchar_t *expression,
                                    const wchar_t *function,
                                    const wchar_t *file,
diff -ru a/lib/gl/msvc-inval.c b/lib/gl/msvc-inval.c
--- a/lib/gl/msvc-inval.c	2014-08-04 20:41:03.938286457 +0200
+++ b/lib/gl/msvc-inval.c	2014-08-04 20:41:40.138286520 +0200
@@ -28,7 +28,7 @@

 # if MSVC_INVALID_PARAMETER_HANDLING == DEFAULT_HANDLING

-static void cdecl
+static void __cdecl
 gl_msvc_invalid_parameter_handler (const wchar_t *expression,
                                    const wchar_t *function,
                                    const wchar_t *file,
@@ -45,7 +45,7 @@

 #  if defined _MSC_VER

-static void cdecl
+static void __cdecl
 gl_msvc_invalid_parameter_handler (const wchar_t *expression,
                                    const wchar_t *function,
                                    const wchar_t *file,
@@ -94,7 +94,7 @@
     }
 }

-static void cdecl
+static void __cdecl
 gl_msvc_invalid_parameter_handler (const wchar_t *expression,
                                    const wchar_t *function,
                                    const wchar_t *file,
